week-3/implement/rotate-char.js

```diff
@@ -1,3 +1,21 @@
+const alphabetLowerCase = [..."abcdefghijklmnopqrstuvwxyz"];
+const alphabetUpperCase = [..."ABCDEFGHIJKLMNOPQRSTUVWXYZ"];
+
+function rotateCharacter(letter, shift) {
+  if (alphabetLowerCase.indexOf(letter) + shift >= 26) {
+    return alphabetLowerCase[alphabetLowerCase.indexOf(letter) + shift - 26];
+  } else if (alphabetUpperCase.indexOf(letter) + shift >= 26) {
+    return alphabetUpperCase[alphabetUpperCase.indexOf(letter) + shift - 26];
+  } else if (alphabetUpperCase.includes(letter)) {
+    return alphabetUpperCase[alphabetUpperCase.indexOf(letter) + shift];
+  } else if (alphabetLowerCase.includes(letter)) {
+    return alphabetLowerCase[alphabetLowerCase.indexOf(letter) + shift];
+  } else if (letter != alphabetLowerCase || letter != alphabetUpperCase) {
+    return letter;
+  }
+}
```

Comment:
Nice one Aisha. In your pull request, you said you have struggled on this exercise, but I can see that you have handled it very well. There are only couple things I wanted to comment about:

1) Just for the sake of making your code more modular, I would take the variables `alphabetLowerCase` and `alphabetUpperCase` inside the function `rotateCharacter(letter, shift)` just because you are only using those variables inside that function.

2) There is actually a way of simplifying this function further more just like you wondered. It involves the usage of the `modulo(%)` operator. But instead of me telling this way, I want you to have a look into(of course if you only have spare time left) what modulo operator is and brainstorm how you can use it to simplify this function. If you still are unsure of it, let me know and we can have a look into it together.

Reply:
thank you I will try the other way to solve the problem. If I get stuck I will reach out to you